mono/mini/debugger-agent.c

```diff
@@ -5595,7 +5595,13 @@ decode_value_internal (MonoType *t, int type, MonoDomain *domain, guint8 *addr,
 				g_free (name);
 				return ERR_INVALID_ARGUMENT;
 			}
-		} else {
+		} else if ((t->type == MONO_TYPE_GENERICINST) && 
+					mono_metadata_generic_class_is_valuetype (t->data.generic_class) &&
+					m_class_is_enumtype (t->data.generic_class->container_class)){
+			err = decode_vtype (t, domain, addr, buf, &buf, limit);
+			if (err != ERR_NONE)
+				return err;
+		} else  {
```

Comment:
Nit: there is an extra space, that triggers an extra diff line.

Reply:
Thanks! Removed!